src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,6 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
+import  java.util.ArrayList;
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.